<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>1171</width>
      <height>751</height>
     </rect>
    </property>
    <widget class="QWidget" name="Main">
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>441</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {


			background-color: #21252d;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="title_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>120</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray</string>
      </property>
      <property name="text">
       <string>Fluorite Scrape</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>521</width>
        <height>651</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {


			background-color: #3C4052;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="url_input">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray;
background-color: #21252d;
border-radius: 25px;</string>
      </property>
      <property name="text">
       <string>Enter Url for website to scrape</string>
      </property>
     </widget>
     <widget class="QFrame" name="text_frame">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>290</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {


			background-color: #595D75;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="text_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>341</x>
        <y>290</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {


			background-color: #272c36;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>320</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray</string>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="images_frame">
      <property name="geometry">
       <rect>
        <x>519</x>
        <y>400</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {


			background-color: #595D75;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="images_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>400</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {


			background-color: #272c36;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>430</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray</string>
      </property>
      <property name="text">
       <string>Images</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="videos_frame">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>510</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {


			background-color: #595D75;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="videos_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>341</x>
        <y>510</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {


			background-color: #272c36;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>540</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray</string>
      </property>
      <property name="text">
       <string>Videos</string>
      </property>
     </widget>
     <widget class="QFrame" name="scrape_frame">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>620</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {


			background-color: #595D75;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="scrape_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <zorder>label_24</zorder>
     <zorder>label_19</zorder>
     <zorder>title_3</zorder>
     <zorder>url_input</zorder>
     <zorder>text_frame</zorder>
     <zorder>label_23</zorder>
     <zorder>label_4</zorder>
     <zorder>images_frame</zorder>
     <zorder>label_20</zorder>
     <zorder>label_6</zorder>
     <zorder>videos_frame</zorder>
     <zorder>label_21</zorder>
     <zorder>label_5</zorder>
     <zorder>scrape_frame</zorder>
    </widget>
    <widget class="QWidget" name="misc_keep">
     <widget class="QFrame" name="manage_buckets_frame_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>240</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="settings_toggle_1_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {


			background-color: #272c36;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>330</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {


			background-color: #272c36;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>430</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {


			background-color: #272c36;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="manage_buckets_frame_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>340</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="settings_toggle_2_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QFrame" name="manage_buckets_frame_4">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>440</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="settings_toggle_3_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>120</y>
        <width>441</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {


			background-color: #21252d;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="title_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>150</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray</string>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>260</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray</string>
      </property>
      <property name="text">
       <string>Faster Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>360</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray</string>
      </property>
      <property name="text">
       <string>Use Download Manager</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>460</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray</string>
      </property>
      <property name="text">
       <string>Some other setting</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>230</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
			background-color: #595D75;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>330</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
			background-color: #595D75;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>430</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
			background-color: #595D75;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>521</width>
        <height>471</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
			background-color: #3C4052;
			padding: 12px;
			border-radius: 25px;
			border-bottom: 30px shadow;
        }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="back_button_frame_8">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>580</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{

	background-color: rgb(60, 64, 82);
	font: 16pt &quot;Montserrat&quot;;
	background-color: 462e2e;
	border-radius: 15px;
}
QPushButton:pressed:hover{
	background-color: rgb(134, 88, 88);
	padding: 12px;
	border-radius: 25px;
}

QPushButton:hover {
	
	background-color: rgb(66, 134, 132);
	padding: 12px;
	border-radius: 25px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="return_home_layout_settings">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <zorder>label_18</zorder>
     <zorder>label_17</zorder>
     <zorder>label_16</zorder>
     <zorder>label_15</zorder>
     <zorder>manage_buckets_frame_2</zorder>
     <zorder>label_11</zorder>
     <zorder>label_12</zorder>
     <zorder>label_13</zorder>
     <zorder>manage_buckets_frame_3</zorder>
     <zorder>manage_buckets_frame_4</zorder>
     <zorder>label_14</zorder>
     <zorder>title_2</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>back_button_frame_8</zorder>
    </widget>
    <widget class="QWidget" name="na10"/>
    <widget class="QWidget" name="na9"/>
    <widget class="QWidget" name="na1"/>
    <widget class="QWidget" name="na"/>
    <widget class="QWidget" name="na8"/>
    <widget class="QWidget" name="na3"/>
    <widget class="QWidget" name="na4"/>
    <widget class="QWidget" name="na5"/>
    <widget class="QWidget" name="na6"/>
    <widget class="QWidget" name="na7"/>
    <widget class="QWidget" name="na2"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource_file_qt.qrc"/>
 </resources>
 <connections/>
</ui>
